ocs_ci/ocs/resources/pod.py

```diff
@@ -203,7 +203,7 @@ def exec_ceph_cmd(self, ceph_cmd, format='json-pretty'):
 
         # For some commands, like "ceph fs ls", the returned output is a list
         if isinstance(out, list):
-            return [item for item in out if item]
+            return [item for item in out]
```

Comment:
ok this is just copy of the out list here, this case I guess we can remove completely this line and line above and just return out on line 207 as we will return list anyway. Or do I miss something here?

Reply:
and if we return list in some cases we should put it to returns section in the docstring. So comment in line 204 should be moved there to docstring.